README.md

```diff
@@ -32,26 +32,33 @@ Stay tuned!
-| - [Jakarta Restful WS](http/restful-ws) |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|          - [Basic](http/basic)          |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|           - [Spring](spring)            |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|           - [http4k][http4k]<sup>†</sup>|                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|            JSON Event Format            |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|    - [Jackson](formats/json-jackson)    |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|            Protobuf Event Format        |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|            - [Proto](formats/protobuf)  |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|     [Kafka Protocol Binding](kafka)     |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
-|          MQTT Protocol Binding          |                          :x:                          |                          :x:                          |
-|          NATS Protocol Binding          |                          :x:                          |                          :x:                          |
-|                Web hook                 |                          :x:                          |                          :x:                          |
+|                                           | [v0.3](https://github.com/cloudevents/spec/tree/v0.3) | [v1.0](https://github.com/cloudevents/spec/tree/v1.0) |
+| :---------------------------------------- | :---------------------------------------------------: | :---------------------------------------------------: |
+| Core                                      |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
+|                                           |                                                       |                                                       |
+| **Protocol Bindings**                     |                                                       |                                                       |
+| AMQP                                      |                          :x:                          |                          :x:                          |
+| - _[Proton](amqp)_                        |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
```

Comment:
Can you replace the `x` with `heavy_check_mark` here since there is a binding implementation?

```suggestion
| AMQP                                      |                          :heavy_check_mark:                          |                          :heavy_check_mark:                          |
| - _[Proton](amqp)_                        |                  :heavy_check_mark:                   |                  :heavy_check_mark:                   |
```